wgpu/Cargo.toml

```diff
@@ -117,6 +117,13 @@ fragile-send-sync-non-atomic-wasm = [
     "wgpu-types/fragile-send-sync-non-atomic-wasm",
 ]
 
+## Replaces AtomicU64.
+portable-atomic = [
+    "dep:portable-atomic",
+    "wgpu-core/portable-atomic",
+    "wgpu-hal/portable-atomic",
+]
```

Comment:
`wgpu` crate can depend on `portable-atomic` conditioned only on `cfg(not(target_has_atomic = "64"))`. Only core and hal need to conditioned on both a feature and `cfg(not(target_has_atomic = "64"))`

Reply:
I tried to change `wgpu` as requested but was not able to propagate features in a `cfg` specific way...

This whole advanced feature/depencency management is a bit over my head.
Can you do the various `Cargo.toml` changes yourself? (maintainers are allowed to edit this PR)

Reply:
Yeah, sure thing, thanks for giving it a shot :)